modules/Movecraft/src/main/resources/localisation/movecraftlang_cz.properties

```diff
@@ -35,56 +35,31 @@ Detection\ -\ Craft\ too\ small=Detekce selhala\! Stroj byl prilis maly. Minimum
 Detection\ -\ Craft\ too\ large=Detekce selhala\! Stroj byl prilis velky. Maximum pro tento typ je %d
 Detection\ -\ Forbidden\ block\ found=Detekce selhala\! Ve stroji byl nalezen zakazany blok.
 Detection\ -\ Failed\ -\ Already\ commanding\ a\ craft=Uz jeden stroj ridis\!
-Detection\ -\ Failed\ Craft\ is\ already\ being\ controlled=Stroj uz je nekym ovladan\!
 Detection\ -\ Successfully\ piloted\ craft=Uspesne ridis stroj\!
 Rotation\ -\ Craft\ is\ obstructed=Stroj uvizl
-Rotation\ -\ Craft\ Collision=Kolize stroje
 Translation\ -\ Failed\ Craft\ hit\ height\ limit=Stroj dosahl limitu vysky.
 Translation\ -\ Failed\ Craft\ is\ obstructed=Stroj se nemuze pohnout, protoze jeho cesta je ucpana\!
 Translation\ -\ Failed\ Craft\ out\ of\ fuel=Stroji doslo palivo\!
 Translation\ -\ Failed\ Craft\ hit\ minimum\ height\ limit=Stroj dosahl minimalniho limitu vysky
-Translation\ -\ Failed\ Craft\ over\ water=Tento stroj se nemuze pohybovat nad vodou\!
-Translation\ -\ Craft\ collision=Kolize stroje
-Internal\ -\ Error\ -\ Proccessor\ thread\ encountered\ an\ error=Async uloha vygenerovala vyjimku.
+Internal\ -\ Error\ -\ Processor\ thread\ encountered\ an\ error=Async uloha vygenerovala vyjimku.
 Startup\ -\ Number\ of\ craft\ files\ loaded=Nacteno %d souboru stroju
-Startup\ -\ Error\ parsing\ CraftType\ file=Chyba pri parsovani souboru TypStroje z adresare: %s
 Startup\ -\ Error\ -\ Reload\ error=Movecraft je nekompatibilni s prikazem reload. Movecraft se vypnul a bude restartovan az s restartem serveru.
 Startup\ -\ Error\ -\ Disable\ warning\ for\ reload=Pokud chcete pouzit prikaz reload a Movecraft, muzete tuto kontrolu vypnout v config.yml nastavenim 'safeReload: false'
 Startup\ -\ Enabled\ message=[V %S] byl nacten.
```

Comment:
Not needed for this PR, but I would like to remove internationalization from startup and console messages.  Unfortunately, we are unable to provide translations for all of our wiki material and as such we expect server owners to have a passable understanding of English console messages.